dist/index.html

```diff
@@ -7,192 +7,226 @@
+        <button type='button' name='button' class='new-data-submit-button' id='sleepSubmitButton'>Submit</button>
+      </form>
+    </section>
+  <section class="main">
+    <section class="main-sidebar-body">
+      <section class="body-sidebar-sidebarContainer">
+        <h2 class="sidebar-header-name" id="sidebarName">Name Name</h2>
+        <div class="sidebar-header-line"></div>
+        <img src="./images/The Rock.jpg" class="sidebar-header-userImage"
+          alt="Smiling face of Dwayne 'the rock' Johnson"></img>
+        <div class="sidebar-header-line"></div>
+        <p class="sidebar-header-userInfo" id="userAddress"></p>
+        <div class="sidebar-header-line"></div>
+        <p class="sidebar-header-userInfo" id="userEmail"></p>
+        <div class="sidebar-header-line"></div>
+        <p class="sidebar-header-userInfo" id="userStridelength"></p>
+        <div class="sidebar-header-line"></div>
+        <p class="sidebar-header-userInfo" id="stepGoal"></p>
+        <p class="sidebar-header-userInfo" id="avStepGoal">
+        <p>
+        <div class="sidebar-header-line"></div>
+        <section class="sidebar-body-friendContainer">
+          <p class="sidebar-header-userInfo">Friends</p>
+          <section class="sidebar-friendContainer-listItems">
+            <ul class="card-vertical-list" id="friendList">
               <!-- friend list goes here -->
             </ul>
           </section>
+          <div class="sidebar-header-line"></div>
+          <p class="thisWeek">Keep up the good work! You were increasingly active on these dates:</p>
+          <ul class="card-vertical-list" id="streakListMinutes">
+            <!-- friend list goes here -->
+          </ul>
+        </section>
+      </section>
+    </section>
+    <section class="body-main-infoContainter">
+      <section class="main-column-hydration">
+        <section class="infoContainer-cardContainer-card-horizontal">
+          <div class="card-today-hydration">
+            <p id="hydrationToday"></p>
+          </div>
+          <div class="card-today-hydration">
+            <p id="hydrationAverage"></p>
+          </div>
+        </section>
+        <section class="infoContainer-cardContainer-card-vertical">
+          <div class="card-history-hydration">
+            <p class="thisWeek">Water intake this week:</p>
+            <div class="sidebar-header-line"></div>
+            <ul class="card-vertical-list" id="hydrationThisWeek">
+              <!-- Hydration week list goes here -->
+            </ul>
+          </div>
+          <div class="card-history-hydration">
+            <p class="historicalWeek"></p>
+            <div class="sidebar-header-line"></div>
+            <ul class="card-vertical-list" id="hydrationEarlierWeek">
+              <!-- Hydration week list goes here -->
+            </ul>
+          </div>
         </section>
       </section>
-      <section class="body-main-infoContainter">
```

Comment:
Adding the info containers to the top definitely helped minimize the impact on the grid layout and responsiveness